pkgs/development/python-modules/python-watcherclient/default.nix

```diff
@@ -19,7 +19,7 @@
 
 buildPythonPackage rec {
   pname = "python-watcherclient";
-  version = "4.4.0";
+  version = "4.5.0";
   pyproject = true;
 
   disabled = pythonOlder "3.8";
```

Comment:
meta.homepage should be switched from the barely functionally gitea on opendev.org to https://github.com/openstack/python-watcherclient to allow doing basic things with the repo...

Reply:
Fixed